dashboard/v1.1/src/pages/Monitoring/Matrix.tsx

```diff
@@ -195,7 +195,19 @@ const Element: FC<ElementProps> = ({ timeSeriesPath, showRouteDetails }) => {
         </Badge>
       </TableCell>
       <TableCell style={{ minWidth: 170, fontSize: 16 }} align="center">
-        <Badge color="success">{'UP'}</Badge>
+        {data.monitor === undefined ? (
+          '-'
+        ) : (
+          <>
+            {data.monitor.values === null ? (
+              '-'
+            ) : data.monitor.values[0].value.resStatusCode === 200 ? (
+              <Badge color="success"> UP</Badge>
+            ) : (
+              <Badge color="secondary">DOWN </Badge>
```

Comment:
I guess in mistake, the space before `UP` and after `DOWN` is applied. Please remove them.